pkg/apis/greenhouse/v1alpha1/pluginbundle_types.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PluginBundleSpec defines the desired state of PluginBundle
+type PluginBundleSpec struct {
+	// ClusterSelector is a label selector to select the clusters the plugin bundle should be deployed to.
+	ClusterSelector metav1.LabelSelector `json:"clusterSelector"`
+
+	// Plugins is a list of plugins that should be deployed.
+	Plugins []PluginConfigSpec `json:"plugins"`
+}
+
+// PluginBundleStatus defines the observed state of PluginBundle
+type PluginBundleStatus struct{}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// PluginBundle is the Schema for the pluginbundles API
+type PluginBundle struct {
```

Comment:
naming?

Reply:
A `PluginPreset` contains default _OptionValues_ for a _PluginDefinition_ that is automatically created for a _Cluster_, if this cluster is matched by the _ClusterSelector_, WDYT? 

The Bundle suggests more than one Plugin is contained.

Reply:
We can put the naming to the test in the next weekly

Reply:
I went ahead with the renaming, as there were no strong opinions raised during the weekly.